tce/src/applibs/Scheduler/Algorithms/CopyingDelaySlotFiller.cc

```diff
@@ -73,6 +73,8 @@ using std::vector;
 using namespace TTAProgram;
 using namespace TTAMachine;
 
+IGNORE_COMPILER_WARNING("-Wstrict-overflow")
+
```

Comment:
This from our own code - is there something to actually fix here? Let's not hide potentially useful warnings.

Reply:
The warning came from line 259, you're right that it probably shouldn't be applied to the entire file.  I looked at the for loop and I don't think it can overflow.